packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/FlowPanels.vue

```diff
@@ -86,7 +86,11 @@ import FlowCanvas from './FlowCanvas.vue'
 
 import type { EdgeChange, NodeChange, VueFlowStore } from '@vue-flow/core'
 
-const { readonly, resizable } = defineProps<{
+const { inactive, readonly, resizable } = defineProps<{
+  /**
+   * Whether current component is inactive (e.g. covered by modal).
+   */
+  inactive?: boolean
```

Comment:
Shall we use the `mode` prop introduced in #2512 instead?

Reply:
Yes. I will rebase this PR after we have merged that one.